### ISSUE TITLE
Upcoming Events on Front Page

### ISSUE THREAD

**Issue by walshbr** [scholarslab/scholarslab.org#418]
The logic on the front page to pull out the upcoming / most recent three events is currently sorting on strings and not dates, so it's not working correctly. I'll open a separate issue for that so that we can close the couple other issues for events once this is merged. Also maybe worth noting that, since this is a static site, there isn't really a way to check for the current date unless we wanted to do some javascript-y thing that I'm not aware of. So the current logic just grabs the three events with the latest three timestamps, regardless of whether those events have happened or not. That is, once it was working that is what it would do.

**Comment by walshbr** [comment]
Actually update to #411 should correct this and close this when merged.

**Comment by amandavisconti** [comment]
@walshbr Is this okay to close (I merged your events import PR), or does it still need work? Thank you!

**Comment by walshbr** [comment]
Can be closed!
